sos/report/plugins/openshift_ovn.py

```diff
@@ -20,18 +20,30 @@ class OpenshiftOVN(Plugin, RedHatPlugin):
     profiles = ('openshift',)
 
     def setup(self):
+        all_logs = self.get_option("all_logs")
+
         self.add_copy_spec([
             "/var/lib/ovn/etc/ovnnb_db.db",
             "/var/lib/ovn/etc/ovnsb_db.db",
-            "/var/lib/openvswitch/etc/keys",
-            "/var/log/openvswitch/libreswan.log",
-            "/var/log/openvswitch/ovs-monitor-ipsec.log"
-        ])
-        # Collect ovn interconnect specific files if exists.
+            "/var/lib/openvswitch/etc/keys"
+        ], sizelimit=300)
+
+        # Collect ovn interconnect specific db files if exists.
         self.add_copy_spec([
             "/var/lib/ovn-ic/etc/ovnnb_db.db",
             "/var/lib/ovn-ic/etc/ovnsb_db.db"
-        ])
+        ], sizelimit=300)
+
+        if not all_logs:
+            self.add_copy_spec([
+                "/var/log/openvswitch/libreswan.log",
+                "/var/log/openvswitch/ovs-monitor-ipsec.log"
+            ])
+        else:
+            self.add_copy_spec("/var/log/openvswitch")
```

Comment:
This folder is also collected in the `openvswitch` plugin, so you could have a conflict of collections. It may be better to target the logname and asterisk. or better yet, if the `openvswitch` plugin does get activated on an openshift node, that these items could be added to that plugin?

Reply:
yes, this is mainly to collect ipsec related logs on `ovn`/`openvswitch` setup. 
now collecting only ipsec logs from ovn plugin when --all-logs option is not set.